enhancements/insights/conditional-data-gathering.md

```diff
@@ -0,0 +1,236 @@
+---
+title: conditional-data-gathering
+authors:
+  - "@Sergey1011010"
+reviewers:
+  - "@inecas"
+  - "@tremes"
+  - "@smarterclayton"
+approvers:
+  - "@smarterclayton"
```

Comment:
Unless @smarterclayton has explicitly agreed to be the approver for this design, we probably want to find someone with more time. Maybe someone from @eparis' group?

Reply:
Sure. How do I find a right person?

Reply:
Good question. Maybe start a thread in #forum-arch on slack?